internal/helm/repository/oci_chart_repository_test.go

```diff
@@ -156,13 +158,23 @@ func TestOCIChartRepoisitory_Get(t *testing.T) {
 			version:     ">2.0.0",
 			expectedErr: "could not locate a version matching provided version string >2.0.0",
 		},
+		{
+			name:     "shouldn't error out with trailing slash",
+			version:  "",
+			url:      "oci://localhost:5000/my_repo/",
+			expected: "1.0.0",
+		},
 	}
 
-	url := "oci://localhost:5000/my_repo"
+	testURL := "oci://localhost:5000/my_repo/"
```

Comment:
I think this should have been
```suggestion
	testURL := "oci://localhost:5000/my_repo"
```